### DISCUSSION TITLE
Filtering in a ham radio

### DISCUSSION THREAD

**Discussion by nicbretz** [AlexandreRouma/SDRPlusPlus#1192] [Ideas]
I have a ham radio transceiver, which has this feature called Twin Passband Tuning. In some instances it can really help pull the signal out of the noise. (I assume other radios have something similar). I found a very short [video on yt ](https://www.youtube.com/watch?v=G-HQEmtrNy4)showing the feature.

- It narrows the IF passband width, feature which exists in SDR++
- It slightly shifts the bandwidth within the IF passband
- It selects soft or sharp filter shoulders

I haven't found any information on exactly how they did it, but it sure looks like two bandpass filters. 

I wrote various algorithms for audio plug-in VST's, including filters, but never anything for SDR. I'm thinking of making this my first SDR project once I become familiar with the code.

**Comment by AlexandreRouma** [comment]
Has already be planned for some time. The DSP to do it is already there it's just a question of implementing the GUI.

I don't take feature pull requests so no need to spend your time implementing this.

**Comment by nicbretz** [comment]
Got it. Yes, the GUI is often the roadblock... Thanks for the reply.
